Gemfile

```diff
@@ -8,8 +8,6 @@ group :development do
 end
 
 group :test do
-  # simplecov test formatter and uploader for Coveralls.io
-  gem "coveralls", '~>0.8.23', :require => false
```

Comment:
This has been dead since 2020, and the endpoint raises SSL exceptions now - potentially due to not providing an auth token.